index.html

```diff
@@ -772,36 +773,1269 @@ <h2 id="BaseAudioContext">
           <a><code>AudioContext</code></a> (for real-time rendering) and
           <a><code>OfflineAudioContext</code></a> (for offline rendering).
         </p>
-        <dl title="enum AudioContextState" class="idl">
-          <dt>
-            suspended
-          </dt>
-          <dd>
-            This context is currently suspended (context time is not
-            proceeding, audio hardware may be powered down/released).
-          </dd>
-          <dt>
-            running
-          </dt>
-          <dd>
-            Audio is being processed.
-          </dd>
-          <dt>
-            closed
-          </dt>
-          <dd>
-            This context has been released, and can no longer be used to
-            process audio. All system audio resources have been released.
-            <span class="synchronous">Attempts to create new Nodes on the
-            AudioContext will throw <code>InvalidStateError</code></span>.
-            (AudioBuffers may still be created, through <a href=
-            "#widl-BaseAudioContext-createBuffer-AudioBuffer-unsigned-long-numberOfChannels-unsigned-long-length-float-sampleRate">
-            createBuffer</a>, <a href=
-            "#widl-BaseAudioContext-decodeAudioData-Promise-AudioBuffer--ArrayBuffer-audioData-DecodeSuccessCallback-successCallback-DecodeErrorCallback-errorCallback">
-            decodeAudioData</a>, or the <a>AudioBuffer</a> constructor.)
-          </dd>
-        </dl>
-        <dl title="interface BaseAudioContext : EventTarget" class="idl"
+        <pre class="idl">
+enum <dfn>AudioContextState</dfn> {
+    "suspended",
+    "running",
+    "closed"
+};
+        </pre>
+        <table class="simple" data-dfn-for="AudioContextState" data-link-for=
+        "AudioContextState">
+          <tbody>
+            <tr>
+              <th colspan="2">
+                Enumeration description
+              </th>
+            </tr>
+            <tr>
+              <td>
+                <dfn><code id=
+                "idl-def-AudioContextState.suspended">suspended</code></dfn>
+              </td>
+              <td>
+                <p>
+                  This context is currently suspended (context time is not
+                  proceeding, audio hardware may be powered down/released).
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn><code id=
```

Comment:
You should put the `<code>` outside the dfn (ReSpec will then know to `code` all  matching `<a>`). Also, please remove the `id`s - those are autogenerated by ReSpec and can change.

Reply:
Swapped dfn and code. Removed the most obvious places where we `id`'s.  There are probably others, but I think it's best to fix those up at the end when we find out links don't work correctly anymore.